system/dummy_diag_publisher/config/dummy_diag_publisher.param.yaml

```diff
@@ -0,0 +1,32 @@
+# Description:
```

Comment:
@asana17 Thank you for the PR!
In addition to this modification, we also need to update [tier4/autoware_launch](https://github.com/tier4/autoware_launch/blob/111cf030d9ab5a46f8b5e2bafa17159fdfbdbd42/system_launch/launch/system.launch.xml#L73-L81). Would you do that to see if this new `dummy_diag_publisher` actually works?

Reply:
@kminoda I made new PRs updating [tier4/autoware_launch](https://github.com/tier4/autoware_launch/pull/762) and [tier4/aip_launcher](https://github.com/tier4/aip_launcher/pull/101). I add test details to this PR (https://github.com/autowarefoundation/autoware.universe/pull/2745#issue-1556128901).